src/OpenIddict.Client/OpenIddictClientModels.cs

```diff
@@ -714,6 +720,11 @@ public sealed record class RefreshTokenAuthenticationResult
         /// </summary>
         public required string AccessToken { get; init; }
 
+        /// <summary>
+        /// Gets or sets the access token expiration date, if available.
```

Comment:
```suggestion
        /// Gets or sets the expiration date of the access token, if available.
```

(for consistency with the comment used for `ProcessAuthenticationContext.BackchannelAccessTokenExpirationDate`)